packages/beacon-node/src/metrics/metrics/lodestar.ts

```diff
@@ -1405,6 +1405,34 @@ export function createLodestarMetrics(
       }),
     },
 
+    builderHttpClient: {
+      requestTime: register.histogram<"routeId">({
+        name: "vc_builder_http_client_request_time_seconds",
```

Comment:
@tuyennhv is it correct that these metrics are prefixed with `vc_`? I assumed this was the prefix used for validator client metrics

Reply:
oh yes that's my bad, thanks @nflaig . Will create a PR to fix.